llvm/lib/SYCL/VXXIRDowngrader.cpp

```diff
@@ -199,9 +199,9 @@ struct VXXIRDowngrader : public ModulePass {
   }
 
   /// Remove Freeze instruction because v++ can't deal with them.
-  /// FIXME: This is not a safe transformation but since LLVM survived with bugs
-  /// caused by absence of freeze for many years, so I guess it is good enough
-  /// for a prototype.
+  /// This is not a safe transformation but since llvm survived with bugs cause
+  /// by absence of freeze for many years, so i guess its its good enough for a
+  /// prototype
```

Comment:
Curious to see this regression.

Reply:
Actually I had to revert the commit that includes this correction, and planned to have a second check on what differs before opening an "actual" PR on triSYCL/SYCL...